R/Rules-class.R

```diff
@@ -297,6 +297,28 @@ validObject(NextBestMinDist(0.1))
 
 ## ============================================================
 
+#' Class for Next Best Dose based on Information Theory.
+#'
+#'
+#' @slot target target DLT probability.
+#' @slot asymmetry value of the asymmetry exponent
+#'
+#' @export
+#' @aliases NextBestInfTh
+.NextBestInfTh <- setClass(Class = "NextBestInfTh",
```

Comment:
I wonder about the name, how about either expanding the "Theory" Bit or just saying "Information" ? I feel the "Th" is a bit too short to know what it could mean